### PR TITLE
Fixes #32872 - return org and loc options to all hammer commands

### PR DESCRIPTION
**Pull request by yifatmakias**

### CODE REVIEW

File: test/functional/architecture_test.rb [hunk 1]
```diff
@@ -24,26 +24,6 @@
       result = run_cmd(@cmd)
       assert_cmd(expected_result, result)
     end
-
-    it 'should run list command with defaults' do
```
Comment (ofedoren) [inline test/functional/architecture_test.rb:28]: Why are you removing tests like that? I find them pretty useful. Although I'd change the version `2.1` to the latest one.

File: test/unit/architecture_test.rb [hunk 2]
```diff
@@ -17,8 +17,6 @@
     context "parameters" do
       it_should_accept "no arguments"
       it_should_accept_search_params
-      it_should_fail_with 'organization param', ['--organization-id=1']
-      it_should_fail_with 'location param', ['--location-id=1']
```
Comment (ofedoren) [inline test/unit/architecture_test.rb:21]: Actually we could leave these tests as well, but test for `it_should_accept` instead :)

**Review by ofedoren — COMMENTED** [review]
Thanks, @yifatmakias for patience and dealing with this issue! :rose:

Few inline comments though...

File: test/functional/architecture_test.rb (hunk 1 above)
Comment (yifatmakias) [reply test/functional/architecture_test.rb:28]: Yes I agree, I think the API calls are different now but I will figure out the correct calls and fix the tests.

File: test/unit/architecture_test.rb (hunk 2 above)
Comment (yifatmakias) [reply test/unit/architecture_test.rb:21]: Good idea I will do that.

**Review by ofedoren — APPROVED** [review]
Thanks, @yifatmakias, LGTM :rose:

**Pull request merged**